cmd/draft/connect.go

```diff
@@ -92,8 +129,10 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 		os.Exit(0)
 	}()
 
-	<-stop
-	return nil
+	for {
+		fmt.Fprintf(cn.out, connectionMessage)
+		time.Sleep(10 * time.Second)
```

Comment:
I noticed that `draft connect` will continually spew the "Your connection is still active" message, so much that the majority of my logs are purely `draft connect` messages. Can we change the frequency to once every 5 minutes by default?

A future enhancement might be to display the message based on the number of logs being spewed out to the screen, but that kinda depends on the screen size.

Reply:
Sure, will update to 5 minutes.